client/src/components/Admin/Categories/AddCategory.js

```diff
@@ -27,7 +27,7 @@ const AddCategory = ({
   const handleSearch = (query) => {
     setIsLoading(true);
     axios
-      .get(`/search-categories?search=${query}`)
+      .get(`/categories/parent-categories?search=${query}`)
```

Comment:
this url could be `/categories/search?q=product` right? 
categories and parent-categories is repeating words

Reply:
here we are getting items for typeahead while adding category. so I kept parent-categories to understand it better